### PR TITLE
Практикум 2

### PR DESCRIPTION
**Pull request by thevlad0**
Факултетен номер: 0MI0600204

### CODE REVIEW

File: SI/Sem.02/Pract.02/Task7/Task7.cpp [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+using namespace std;
+
+int main()
+{
+    char letter;
+    cout << "Enter a letter: ";
+    cin >> letter;
+    if (letter >= 'a' && letter <= 'z')
+        cout << "The big letter is " << (char)(letter - 32) << endl;
```
Comment (GeorgiTerziev02) [inline SI/Sem.02/Pract.02/Task7/Task7.cpp:11]: We actually have not learnt casting yet :), just increment/decrement the letter before the cout

File: SI/Sem.02/Pract.02/Task8/Task8.cpp [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+using namespace std;
+
+int main()
+{
+   int num;
+   cout << "Arabic numeral: ";
+   cin >> num;
+   
+   cout << "Roman numeral: ";
+   for (size_t i = 0; i < num/10; i++)
```
Comment (GeorgiTerziev02) [inline SI/Sem.02/Pract.02/Task8/Task8.cpp:12]: The program should work only for ONE DIGIT numbers.
Also, check your case for the following input: 4.

File: SI/Sem.02/Pract.02/Task12/Task12.cpp [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+                    cout << "Invalid input" << endl;
+                    return 1;
+                }
+                else day++;
+                break;
+
+            default:
+                if(day == 30)
+                {
+                    if(month == 12)
```
Comment (GeorgiTerziev02) [inline SI/Sem.02/Pract.02/Task12/Task12.cpp:60]: This check is useless here

File: SI/Sem.02/Pract.02/Task13/Task13.cpp [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    if (a < 1000 || a > 9999)
+    {
+        cout << "Invalid input" << endl;
+        return 1;
+    }
+    else
+    {
+        int a1, a2;
+        
+        a1 = (a/1000)*10 + ((a%1000)%100)%10;
```
Comment (GeorgiTerziev02) [inline SI/Sem.02/Pract.02/Task13/Task13.cpp:19]: Refactor line 19 and line 20, there is much simpler solution to the the digit
Line 19 just directly use %10 in the end

**Review by GeorgiTerziev02 — CHANGES_REQUESTED** [review]
Looks good but there are small things that can be improved.
Good job for task 9 and task 11.

**Comment by Justsvetoslavov** [comment]
Review the new written PR rules and open new PRs if you would like (also check which ones are already merged ,as they will be duplicates).
[You can view them here](https://github.com/Justsvetoslavov/Introduction_to_programming)

**Pull request closed without merging**